.github/workflows/linux_ci.yml

```diff
@@ -0,0 +1,36 @@
+on: [push]
+
+name: Linux CI
+
+jobs:
+  check:
+    name: Compile and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install gtk3 and dependencies.
+        run: sudo apt-get update && sudo apt install menu libcurl4-openssl-dev libelf-dev libdw-dev binutils-dev libgtk-3-dev libmount-dev libsoup2.4-dev libxtst-dev at-spi2-core openbox
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      
+      - name: Check Code
```

Comment:
If you run the check code here, please run clippy here as well, that will be one less file to keep around.

Reply:
Okay, Clippy should now run here as well. This has been reflected in the mac_ci as well for consistency.